perf-site-replication.go

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright (c) 2015-2023 MinIO, Inc.
+//
+// This file is part of MinIO Object Storage stack
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+//
+
+package madmin
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/url"
+	"time"
+)
+
+// SiteNetPerfNodeResult  - stats from each server
+type SiteNetPerfNodeResult struct {
+	Endpoint string `json:"endpoint"`
+	TX       uint64 `json:"tx"`
+	RX       uint64 `json:"rx"`
+	Error    string `json:"error,omitempty"`
```

Comment:
Don't we need a time indication here, so we know the time frame spent transferring RX/TX?

I feel that should be added for completeness, so we don't have to rely on the query to parse the result.

Reply:
Don't we measure ping latency? Or is that out of scope?

Reply:
Also add the number of concurrent connections per peer.